habitat-lab/habitat/utils/humanoid_utils.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import glob
+import pickle as pkl
+
+import magnum as mn
+import numpy as np
+import pybullet as p
+
+joint_names = [
```

Comment:
For which source? Let's be specific here.

Reply:
Clarified